src/user/proposals/ProposalDiscussion/DiscussionContent/DiscussionComments/DiscussionComments.js

```diff
@@ -1,32 +1,78 @@
 import React, { Component } from "react";
 import userIcon2 from "../../../../../images/userIcon2.jpg";
-import { Form, ListGroup } from "react-bootstrap";
+import { Form, ListGroup, Button } from "react-bootstrap";
 import "./DiscussionComments.scss";
 
 class DiscussionComments extends Component {
   constructor(props) {
     super(props);
-    this.state = {};
+    this.state = {
+      commentContent: "",
+      commentItems: this.props.commentItems,
+    };
   }
 
+  handleComment = (text) => {
+    fetch("http://localhost:5000/proposal/comment", {
```

Comment:
Same here, make it as seperate action, also use axios as it's already used in most of the part of integration, It's stable as well.